10-JavaScript/testing.md

```diff
@@ -23,96 +22,119 @@ In short, most interesting JavaScript code is full of *external dependencies*. M
 
 To get around this, there are basically two strategies to follow:
 - Use a technology like [Selenium](http://www.seleniumhq.org/) to *mock* the DOM
 - Build your application so that as much of the business logic as possible can run without the DOM
 
-Since our remaining time together is short, we'll use the second technique. It will be very reminiscent of model testing in Rails, and in fact we will focus our JavaScript testing on models. The testing techniques we learn here will also transfer directly to any command-line JavaScript programs you write, including something like a Node-Express server.
+We'll start testing using the second technique. It will be very reminiscent of model testing in Rails, and in fact we will focus our JavaScript testing on business logic methods. The testing techniques we learn here will also transfer directly to any command-line JavaScript programs you write, including something like a Node-Express server.
 
 As an aside, Selenium is a *really* cool technology, we just don't have the time to do it justice. Maybe it would make a good piece of a capstone project...?
 
 ### Setup
-We will utilize the same basic application structure that we have used for our Backbone applications. In fact, the project we'll be working with is a complete Backbone SPA. To let us focus on the process of writing tests with Jasmine rather than implementing some bit of complex logic, we'll be using a Backbonified version of our good old Scrabble game.
+We will start writing tests with a sample whiteboarding problem.  
 
-1. Clone this repo: `https://github.com/AdaGold/scrabble-jasmine-baseline`
+1. Clone this repo: `https://github.com/AdaGold/pangram`
 1. `npm install`
 1. `npm start`
 
-Spend some time playing with the site. What models, collections and views do you think there might be? What functionality is still missing?
+Examine the README.  
 
-Now look through the code, and confirm or rebuke your suspicions. Pay close attention to the `spec` directory, which is where the tests live - we'll dive into it a little later. To run the tests, type `npm test` at the console.
+**Question**: In testing a Pangram, what edge-cases would you have?  
 
-Similar to Rails, the `spec` directory has the same structure as the `src` directory. Each file `src/dir/name.js` may have a corresponding `spec/dir/name.spec.js`.
+Now look through the code in the `pangram.spec.js` file, and see if the tests described match your predictions. To run the tests, type `npm test` at the console.
 
-Because we're running tests via `npm`, our tests have access to all our WebPack infrastructure, and `import` and `export` statements will work about as you'd expect.
+Similar to Rails, the `spec` directory has the same structure as the `src` directory. Each file `src/dir/name.js` may have a corresponding `spec/dir/name.spec.js`.
```

Comment:
There is no `src` directory, so if you want to see this parallel this should be added to the file structure in pangram